meinberlin/assets/scss/components/_hero_unit.scss

```diff
@@ -0,0 +1,40 @@
+.hero-unit {
+    position: relative;
+    min-height: 25em;
```

Comment:
I am not a big fan of magic numbers, but I guess in this specific context they are fine.